### ISSUE TITLE
Exception: SNlM0e value not found in response. Check __Secure-1PSID value.

### ISSUE THREAD

**Issue by altkriz** [AbhiWisdom/Bard-Telegram-Bot#1]
I tried everything you told me but this error happens
Exception has occurred: Exception
SNlM0e value not found in response. Check __Secure-1PSID value.
  File "C:\Users\altkr\Desktop\telegram_python bot\main.py", line 14, in <module>
    chatbot = Chatbot(token)
              ^^^^^^^^^^^^^^
Exception: SNlM0e value not found in response. Check __Secure-1PSID value.

**Comment by AbhiWisdom** [comment]
Yes now this trick got patched .. if you need more help related these types
of bot then join our telegram channel @abhibots

**Comment by m23xp** [comment]
Can I still use the method? If the answer is no, is there **another way**?

**Comment by AbhiWisdom** [comment]
You can use gemini api , its free to use

**Comment by m23xp** [comment]
Do I need to modify the code?

**Comment by AbhiWisdom** [comment]
You need to write your own code ..

If you wants to see my other bots must join @abhibots on telegram

**Comment by m23xp** [comment]
I am already a subscriber to the channel and a follower on Telegram, Thank you 🤍

**Comment by AbhiWisdom** [comment]
You're welcome!
